gdal/gcore/gdal_priv.h

```diff
@@ -1181,6 +1181,7 @@ class CPL_DLL GDALRasterBand : public GDALMajorObject
     virtual CPLErr SetStatistics( double dfMin, double dfMax,
                                   double dfMean, double dfStdDev );
     virtual CPLErr ComputeRasterMinMax( int, double* );
+    virtual void SetValidPercent( GUIntBig nSampleCount, GUIntBig nValidCount );
```

Comment:
Please move it in the private: section of the class definition, and remove the virtual keyword

Reply:
Done